<launch>
    <arg name="use_robot" default="false"/>
    <arg name="robot_ip" default=""/>

    <node pkg="isat_robot_control" type="misty_ROS.py" name="mistyROS">
        <param name="use_robot" type="bool" value="$(arg use_robot)" />
        <param name="robot_ip" type="string" value="$(arg robot_ip)"/>
    </node>

</launch>